### PR TITLE
whitespace cleanup

### PR DESCRIPTION
**Pull request by edhartnett**
Part of #318

### CODE REVIEW

File: src/enc_png.c [hunk 1]
```diff
@@ -85,100 +85,76 @@ void user_flush_data(png_structp png_ptr)
...
-*/
 
-    /*bytes=bit_depth/8;*/
-    bytes=*nbits/8;
-    row_pointers=malloc((*height)*sizeof(png_bytep));
-    for (j=0;j<*height;j++) row_pointers[j]=(png_bytep *)(data+(j*(*width)*bytes));
+    /* Put image data into the PNG info structure. */
+    bytes = *nbits / 8;
+    row_pointers = malloc((*height) * sizeof(png_bytep));
+    for (j = 0; j < *height; j++)
```
Comment (AlexanderRichert-NOAA) [inline src/enc_png.c:147]: There should be a space between `*` and `height`

**Comment by AlexanderRichert-NOAA** [comment]
Are you using any automatic formatting tools? And if not would it be worth looking into?

File: src/enc_png.c (hunk 1 above)
Comment (edwardhartnett) [reply src/enc_png.c:147]: No because that's a pointer dereference.

**Comment by edwardhartnett** [comment]
I would be up for using an autoformatter for fortran code.

Currently I use emacs which has a lot of build-in tools for this. BUt I'm trying out VSCode and I don't know yet what that provides...

**Review by edwardhartnett — APPROVED** [review]

**Pull request merged**